src/wrapper/run.rs

```diff
@@ -70,16 +71,8 @@ pub fn run_command_cmd(wrapper: &Wrapper, cmdline: Vec<String>)
     let env = try!(setup::get_environment(cconfig, &wrapper.settings));
     let mut cpath = PathBuf::from(&command);
     let args = args.clone().to_vec();
-    if command.contains("/") {
-    } else {
-        let paths = [
-            "/bin",
-            "/usr/bin",
-            "/usr/local/bin",
-            "/sbin",
-            "/usr/sbin",
-            "/usr/local/sbin",
-        ];
+    if !command.contains("/") {
+        let paths: Vec<_> = DEFAULT_PATH.split(':').collect();
```

Comment:
There is usage of `paths` below:
```
         if !cpath.is_absolute() {
             return Err(format!("Command {:?} not found in {:?}",
                 cpath, &paths));
}
```

Or we can use `DEFAULT_PATH` instead of array.